src/style.css

```
@@ -0,0 +1,150 @@
+html {
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+}
+
+body {
+  margin: 0;
+}
+
+h2,
+p {
+  font-size: 0;
+  margin: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.card {
+  --card-border-color: #f3f3f3;
+  --card-txt-first-color: #060b35;
+  --card-txt-second-color: #616070;
+  width: 200px;
+  height: 404px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  border: 1px solid var(--card-border-color);
+  border-radius: 5px;
```

```suggestion
  border-radius: 5px;
  padding: 32px 16px 16px
```

add padding for your card